testing/tests/test_api.py

```diff
@@ -15,6 +15,30 @@ def test_verification_api(self, testapp):
         assert 200 == ver_res.status_code
         assert ver_res.json['results'][0]['tamper'] > 0.5
 
+    @pytest.mark.usefixtures("testapp")
+    def test_fps_parameters(self, testapp):
+        # test verification with 0 fps
+        ver_res = testapp.post('/verify', json=dict(source='testing/tests/data/master_4s_1080.mp4', orchestratorID='1',
+                                                    renditions=[{'uri': 'testing/tests/data/master_4s_1080.mp4',
+                                                                 'frame_rate': 0}]))
+        assert 200 == ver_res.status_code
+        assert ver_res.json['results'][0]['tamper'] < 0.5
```

Comment:
Isn't the tamper value 0 or 1? If so, any reason to check against 0.5 instead of 0 or 1?

Reply:
No reason currently, fixed.